WinSnitch/build.ps1

```
@@ -0,0 +1,34 @@
+$ErrorActionPreference = "Stop"
+
+$Root = (Get-Item -Path ".\" -Verbose).FullName
+
+if ($BuildBackend) {
+    Write-Host "--- Building Go Backend ---"
+    Set-Location -Path "$Root\backend"
+    go mod tidy
+    $env:GOOS = "windows"
+    $env:GOARCH = "amd64"
+    go build -o "$Root\installer\WinSnitchDaemon.exe" main.go wfp.go
+    if ($?) { Write-Host "Backend built successfully." -ForegroundColor Green }
```

PSScriptAnalyzer (1.24.0)

[warning] File 'build.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)